src/main/drivers/dshot_bitbang_impl.h

```diff
@@ -79,6 +79,9 @@
 #ifdef USE_HAL_DRIVER
 #define BB_GPIO_PULLDOWN GPIO_PULLDOWN
 #define BB_GPIO_PULLUP   GPIO_PULLUP
+#elif defined(AT32F435)
```

Comment:
We should aim to remove any MCU specific defines from the main drivers code.

Reply:
then we need to refactor this file and make it part of the stm32/at32 driver folder.

Reply:
Perhaps not the entire file, just the parts that have specific defines.